csvtotable/convert.py

```diff
@@ -130,9 +130,10 @@ def render_template(table_headers, table_items, **options):
     virtual_scroll = False
     scroll_y = height
 
-    if virtual_scroll_limit != -1 and len(table_items) > virtual_scroll_limit:
-        virtual_scroll = True
-        display_length = -1
+    if virtual_scroll_limit:
```

Comment:
here is the notebook if you want to run it and check it yourself

https://github.com/pyexcel/pyexcel-sortable/blob/master/notebook.ipynb